### ISSUE TITLE
Practical 3 Perform the data classification using the classification algorithm

### ISSUE THREAD

**Issue by RenuSam6964** [RenuSam6964/TY#2]
from sklearn.datasets import load_iris
from sklearn.model_selection import train_test_split
from sklearn.preprocessing import StandardScaler
from sklearn.svm import SVC
from sklearn.metrics import accuracy_score,classification_report,confusion_matrix

iris = load_iris()
x = iris.data
y = iris.target
x_train,x_test,y_train,y_test = train_test_split(x,y,test_size = 0.2, random_state = 42)

scalar=StandardScaler()
x_train = scalar.fit_transform(x_train)
x_test = scalar.transform(x_test)

svm = SVC(kernel = 'rbf', C = 1)
svm.fit(x_train, y_train)
y_pred = svm.predict(x_test)

accuracy = accuracy_score(y_test, y_pred)
print(f'Accuracy: {accuracy:.3f}')
print('Classification Report')
print(classification_report(y_test, y_pred))
print('Confusion Matrix')
print(confusion_matrix(y_test, y_pred))
